docs/getting-started/croeseid-testnet.md

```diff
@@ -132,8 +132,8 @@ Follow the below optional steps to enable state-sync.
 - For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
 
   ```bash
-  $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
-  BLOCK_HEIGHT=$((LASTEST_HEIGHT - 1000)); \
+  $ LATEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
```

Comment:
```suggestion
  $ LATEST_HEIGHT=$(curl -s  https://testnet-croeseid-3.crypto.org:26657/block  | jq -r .result.block.header.height); \
```

Reply:
@zanglang Oh I found that it should be connecting to the latest public fullnode of  `testnet-croeseid-3` which is https://testnet-croeseid-3.crypto.org:26657/block